snapshot-labs/sx-monorepo
# fix: Improve topic component styles

### Summary

- Remove border-bottom for reply button
- Add more space between content and like, comment icons

### How to test

1. Go to any discussion page http://localhost:8080/#/s:safe.eth/proposal/0x2c2ad686e94fd97903de8ee520b90d400c98a1985e31b21416da095b28373f18/discussion
2. Check for new changes
